packages/shared/src/components/auth/CodeVerificationForm.tsx

```diff
@@ -11,7 +11,6 @@ import { AuthEventNames } from '../../lib/auth';
 import AnalyticsContext from '../../contexts/AnalyticsContext';
 import AuthForm from './AuthForm';
```

Comment:
Just something that I found very confusing when reviewing the pervious and this PR..the file names 😭 

EmailCodeVerification, CodeVerificationForm

Reply:
I know! Sole reason this was introduced wrongly 🤯
Really want to prio some work on auth to abstract/redo some stuff and fix naming/imports.